osrm/core.py

```diff
@@ -187,9 +187,9 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
 
         url.extend([
             '{},{}'.format(coord_dest[0], coord_dest[1]),
-            "?overview={}&steps={}&alternatives={}&geometries={}".format(
+            "?overview={}&steps={}&alternatives={}&geometries={}&continue_straight={}".format(
                  overview, str(steps).lower(),
-                 str(alternatives).lower(), geom_request)
+                 str(alternatives).lower(), geom_request, continue_straight)
```

Comment:
Could you also add the `annotations` option here ? 
And also add  the `continue_straight` option a few lines below (https://github.com/ustroetz/python-osrm/blob/master/osrm/core.py#L196) so the two branches of the condition will use the same options when building the url ?